addons/repair/models/repair.py

```diff
@@ -111,6 +111,13 @@ class Repair(models.Model):
         readonly=True, required=True, index=True,
         default=lambda self: self.env.company)
     sale_order_id = fields.Many2one('sale.order', 'Sale Order', copy=False, help="Sale Order from which the product to be repaired comes from.")
+    picking_id = fields.Many2one(
+        'stock.picking', 'Return',
```

Comment:
check_company=True,

Reply:
We can link repair orders to SOs with another company_id :thinking: do we not want to do the same for repairs? Or should I change the SO to check_company as well?.

Reply:
Done, also added/fixed for SOs.